frontend/javascripts/admin/voxelytics/ai_model_list_view.tsx

```diff
@@ -0,0 +1,151 @@
+import _ from "lodash";
+import React, { useEffect, useState } from "react";
+import { SyncOutlined } from "@ant-design/icons";
+import { Table, Button, Modal } from "antd";
+import { getAiModels } from "admin/admin_rest_api";
+import Toast from "libs/toast";
+import { AiModel } from "types/api_flow_types";
+import FormattedDate from "components/formatted_date";
+import { formatUserName } from "oxalis/model/accessors/user_accessor";
+import { useSelector } from "react-redux";
+import { OxalisState } from "oxalis/store";
+import { JobState } from "admin/job/job_list_view";
+import { Link } from "react-router-dom";
+
+export default function AiModelListView() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [aiModels, setAiModels] = useState<Array<AiModel>>([]);
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
```

Comment:
I think is it nowherer checked whether the active user is a admin / super user. But this feature should only be available to admins, correct?

Thus, this view should also only be visible to superusers.